$NetBSD$

Compilation on NetBSD-4.99.7/amd64 fails if these are not
commented out. Better solution happily accepted...

--- gemrb/plugins/ACMImporter/general.h.orig	2005-02-23 22:20:52.000000000 +0000
+++ gemrb/plugins/ACMImporter/general.h
@@ -25,7 +25,9 @@
 // Interplay ACM signature
 #define IP_ACM_SIG 0x01032897
 
+#if 0
 #pragma pack (push, 1)
+#endif
 struct ACM_Header {
 	unsigned int signature;
 	unsigned int samples;
@@ -34,6 +36,8 @@ struct ACM_Header {
 	unsigned short levels : 4;
 	unsigned short subblocks : 12;
 };
+#if 0
 #pragma pack (pop)
+#endif
 
 #endif
